manifest.toml

```
@@ -43,8 +43,8 @@ ram.runtime = "0M"
     [resources.sources]
 
         [resources.sources.main]
-        url = "https://github.com/jellyfin/jellyfin-vue/archive/f1efc3cd0923d09af69debea1e19a08b24310afc.tar.gz"
-        sha256 = "7f3a81a44153b456012bb0c2a0d404eb0b4ac0db1c56768cef7e37ed4eb52c6f"
+        url = "https://github.com/jellyfin/jellyfin-vue/archive/a131e130ce933be17e84bcbffa4e1fd2ae05a6cf.tar.gz"
+        sha256 = "505dd6904e00b56b29a0b8257001b2e87bb51788d888f516585237ee2f24dedc"
 
         autoupdate.upstream = "https://github.com/jellyfin/jellyfin-vue/"
         autoupdate.strategy = "latest_github_commit"
```

```suggestion
        autoupdate.strategy = "latest_github_tag"
```

Wouldn’t it be better to only fetch latest tags instead of commit? As tags are more for "production" release and commits for development, and because there are new commits everyday?